### PR TITLE
English translation - profiles

### PR DESCRIPTION
**Pull request by christpet**
All set - the only thing I'm not sure about is the changed date format.

### CODE REVIEW

File: src/i18n/resources/en/profiles.json [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
   "common": {
-    "sources": "Les sources",
-    "subscriber": "Abonné(e)",
-    "similar_profiles": "Profils similaires",
-    "no_more_contribution": "Pas d'autres contributions",
+    "sources": "Guides",
+    "subscriber": "Subscribers",
+    "similar_profiles": "Similar profiles",
+    "no_more_contribution": "No more contributions",
     "all": "Tous"
```
Comment (gregoirelacoste) [inline src/i18n/resources/en/profiles.json:7]: it miss "all" ?

File: src/i18n/resources/en/profiles.json [hunk 2]
```diff
@@ -1,77 +1,77 @@
...
+      "description_dismoi":"DisMoi lets internet users, media and experts share helpful information directly on the webpages you visit."
     }
   },
   "notice": {
-    "screenshot_alt": "Rendu de la contribution sur {{exampleMatchingUrl}} une fois l'extension installée.",
-    "pined_on": "<0>Message épinglé sur <1>{{exampleMatchingUrl}}</1></0> et d'autres pages web",
-    "since": "Visible depuis le {{date, DD/MM/YYYY}}"
+    "screenshot_alt": "Shows the contribution on {{exampleMatchingUrl}} once the extension is installed.",
+    "pined_on": "<0>Message pinned on <1>{{exampleMatchingUrl}}</1></0> and other webpages",
+    "since": "Since {{date, MM/DD/YYYY}}"
```
Comment (gregoirelacoste) [inline src/i18n/resources/en/profiles.json:75]: Fine !

**Review by gregoirelacoste — CHANGES_REQUESTED** [review]
Nice, All that's missing is "all", thank you !!

File: src/i18n/resources/en/profiles.json (hunk 1 above)
Comment (christpet) [reply src/i18n/resources/en/profiles.json:7]: Ah yes I did!

Comment (christpet) [reply src/i18n/resources/en/profiles.json:7]: Thanks!

Comment (christpet) [inline src/i18n/resources/en/profiles.json:7]:
```suggestion
    "all": "All"
```

**Comment by lutangar** [comment]
@gregoirelacoste  you let unformatted commit messages slip through the pipeline.
I might add an "automated thing" to prevent this to happens unnoticed.

> For reference, this convention is responsible for setting  the extension version number and generates changelog on new releases:
> https://github.com/dis-moi/extension/releases
>
> This is mentionned here and there:
> https://github.com/dis-moi/extension/blob/master/docs/CONTRIBUTING.md#commits
> https://github.com/dis-moi/docs/pull/1/files

@christpet I'd be more confortable if a tech person would hit the merge button as long as we can let human errors go through to the pipe unnoticed.

**Pull request merged**